plugins/log_request_plugin/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
```

Comment:
Can this `log_request_plugin` project be moved from the top level of `plugins` directory to `plugins/operations/c8y` directory to better reflect that it's a c8y-specific operation plugin? It also helps avoid mixing up these operation plugins from the sm-plugins that are already there at this top level. These sm-plugins also need to be grouped together and moved into `plugins/software` directory later on in a different PR to complete that cleanup.

Reply:
I think eventually we will have to organize the plugins somehow, maybe even move them to a separate repository, but I would rather have a specific task and put some thought about future organisation of those so we don't end up in the same situation we had with core tedge before..

@didier-wenzek  What are your thoughts

Reply:
I'm happy with the current almost flat hierarchy !

* @albinsuresh Your proposal remembers me what we had a few months ago when it was not obvious to find a crate lost in a deep hierarchy.
* I see no problem in having in the same directory crates with different usages.